go/vt/vtgate/planbuilder/dml.go

```diff
@@ -179,15 +179,15 @@ func buildDMLPlan(vschema ContextVSchema, dmlType string, stmt sqlparser.Stateme
 	return edml, ksidVindex, ksidCol, nil
 }
 
-func generateDMLSubquery(where *sqlparser.Where, orderBy sqlparser.OrderBy, limit *sqlparser.Limit, table *vindexes.Table, ksidCol string) string {
+func generateDMLSubquery(tblExpr sqlparser.TableExpr, where *sqlparser.Where, orderBy sqlparser.OrderBy, limit *sqlparser.Limit, table *vindexes.Table, ksidCol string) string {
```

Comment:
`table` is an unused argument now, isn't it?

Reply:
it is used to get vindex columns for projection columns